controllers/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -651,6 +673,16 @@ func ExtractStatusMetadata(instance *toolchainv1alpha1.ToolchainStatus) []*Compo
 				Message:       cond.Message,
 			})
 		}
+
+		cond, found = condition.FindConditionByType(instance.Status.RegistrationService.RevisionCheck.Conditions, toolchainv1alpha1.ConditionReady)
+		if found && cond.Status != corev1.ConditionTrue {
+			result = append(result, &ComponentNotReadyStatus{
+				ComponentType: "Registration service",
+				ComponentName: "Revision Check",
```

Comment:
same here, `ComponentType` and `ComponentName` looks inverted

Reply:
inverted , please see: https://github.com/codeready-toolchain/host-operator/pull/836/commits/54d573742cad4df097d7652a5ae06a356182acf0